man/nss-mymachines.xml

```diff
@@ -35,12 +35,20 @@
     <para><command>nss-mymachines</command> is a plug-in module for the GNU Name Service Switch (NSS) functionality of
     the GNU C Library (<command>glibc</command>), providing hostname resolution for the names of containers running
     locally that are registered with
-    <citerefentry><refentrytitle>systemd-machined.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.  The
+    <citerefentry><refentrytitle>systemd-machined.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>. The
     container names are resolved to the IP addresses of the specific container, ordered by their scope. This
-    functionality only applies to containers using network namespacing.</para>
-
-    <para>The module also resolves user and group IDs used by containers to user and group names indicating the
-    container name, and back. This functionality only applies to containers using user namespacing.</para>
+    functionality only applies to containers using network namespacing (see the description of
+    <option>--private-network</option> in
+    <citerefentry><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>).
+    Note that the name that is resolved is the one registered with <command>systemd-machined</command>, which
+    may be different than the hostname configured inside of the container.</para>
+
+    <para>The module also provides name resolution for user and group identifiers mapped to containers. All names from
+    the range allocated to a given container <replaceable>container</replaceable> are exposed on the host with the
+    <literal>vu-<replaceable>container</replaceable></literal> and
```

Comment:
should maybe say `<literal>vu-<replaceable>container</replaceable>-<replaceable>uid</replaceable></literal>`?